app/client/src/components/pages/Community/components/tabs/Monitoring.js

```diff
@@ -116,39 +107,45 @@ type StationGroups = {
   },
 };
 
-type State = {
-  prevMonitoringLocationData: MonitoringLocationData,
-  monitoringLocationToggles: { [label: string]: boolean },
-  monitoringStationGroups: StationGroups,
-  allMonitoringStations: Array<Station>,
-  displayedMonitoringStations: Array<Station>,
-  allToggled: boolean,
-  sortBy: string,
-};
+function Monitoring({ esriModules, infoToggleChecked }: Props) {
+  const {
+    monitoringLocations,
+    monitoringGroups,
+    showAllMonitoring,
+    setMonitoringGroups,
+    monitoringStationsLayer,
+    setShowAllMonitoring,
+    watershed,
+  } = React.useContext(LocationSearchContext);
+
+  const [
+    prevMonitoringLocationData,
+    setPrevMonitoringLocationData,
+  ] = React.useState({});
```

Comment:
I think it would be better to go ahead and include the `MonitoringLocationData` type (lines 94 - 101 in the old code). You can use it on the state like this

```suggestion
  const [
    prevMonitoringLocationData,
    setPrevMonitoringLocationData,
  ] = React.useState<MonitoringLocationData>({});
```